kedro/pipeline/node.py

```diff
@@ -599,7 +599,7 @@ def node(
     outputs: Union[None, str, List[str], Dict[str, str]],
     *,
     name: str = None,
-    tags: Iterable[str] = None,
+    tags: Union[str, Iterable[str]] = None,
```

Comment:
This should always have been this way so that `node` and `Node` match identically.